docs/preprocess.rst

```diff
@@ -125,7 +135,44 @@ This pipeline can be run through the functions saved in ``site_pipeline``. Each
 entry in "process_pipe" key will be used to generate a Preprocess module based
 on the name it is registered to. These entries will then be run in order through
 the processing pipe. The ``process`` function is always run before the
-``calc_and_save`` function for each module.
+``calc_and_save`` function for each module. The ``plot`` function can be run after
+``calc_and_save`` when ``plot: True`` for a module that supports it. Current modules
+with plotting: ``DetBiasFlags``, ``EstimateHWPSS``, ``SSOFootprint``.
+
+Example TOD Pipeline Configuration File
+---------------------------------------
+
+Suppose we want to run an observation-level pipeline that creates a SSO footprint.
+A configuration file for the processing pipeline would look like::
+
+    # Context for the data
+    context_file: 'context.yaml'
+
+    # Plot directory prefix
+    plot_dir: './plots'
+
+    # Metadata index & archive filenaming
+    archive:
+        index: 'preprocess_archive.sqlite'
+        policy:
+            type: 'simple'
+            filename: 'preprocess_archive.h5'
+
+    process_pipe:
+        - name: "sso_footprint"
+          calc:
+            distance: 20
+            nstep: 100
+          save: True
+          plot:
+            wafer_offsets: {'ws0': [-2.5, -0.5],
+                            'ws1': [-2.5, -13],
+                            'ws2': [-13, -7],
+                            'ws3': [-13, 5],
+                            'ws4': [-2.5, 11.5],
+                            'ws5': [8.5, 5],
+                            'ws6': [8.5, -7]}
+            focal_plane: '/so/home/msilvafe/shared_files/sat_hw_positions.npz'
```

Comment:
I still want to get this moved to somewhere more official and don't really want a path to somewhere in my user home on site computing in docs for a public repo.

Reply:
always use fake paths for the documentation.

And I think this is something we could load from Saianeesh's setup? A future issue/upgrade, shouldn't prevent merging.